libdevcore/CommonIO.h

```diff
@@ -77,7 +77,7 @@ std::string memDump(bytes const& _bytes, unsigned _width = 8, bool _html = false
 template <class S, class T> struct StreamOut { static S& bypass(S& _out, T const& _t) { _out << _t; return _out; } };
 template <class S> struct StreamOut<S, uint8_t> { static S& bypass(S& _out, uint8_t const& _t) { _out << (int)_t; return _out; } };
 
-inline std::ostream& operator<<(std::ostream& _out, bytes const& _e) { _out << toHex(_e, 2, HexPrefix::Add); return _out; }
+inline std::ostream& operator<<(std::ostream& _out, bytes const& _e) { _out << "0x" << toHex(_e); return _out; }
```

Comment:
Note that this might make a difference depending on the stream. I think some logging output stream we use weirdly inserts spaces between two elements.